getting started/crowdsale.sol

```diff
@@ -0,0 +1,76 @@
+contract token { mapping (address => uint) public balance; function token() {}  function sendToken(address receiver, uint amount) returns(bool sufficient) {  } }
```

Comment:
I expanded the content into different lines, as requested.

On Tue, Jun 2, 2015 at 11:42 AM, chriseth <notifications@github.com> wrote:

> In getting started/crowdsale.sol
> <https://github.com/ethereum/dapp-bin/pull/22#discussion_r31529785>:
>
> > @@ -0,0 +1,76 @@
> > +contract token { mapping (address => uint) public balance; function token() {}  function sendToken(address receiver, uint amount) returns(bool sufficient) {  } }
>
> contract token {
>  mapping (address => uint) public balance;
>  function token();
>  function sendToken(address receiver, uint amount) returns(bool sufficient);
> }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/dapp-bin/pull/22/files#r31529785>.
>



-- 

Alex Van de Sande
UX Designer